### ISSUE TITLE
[Bug]: Comment Downloading - AttributeError: 'NoneType' object has no attribute 'replace'

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#429]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

Linux

## Describe the bug
During download post-processing (comments) the below error occurred and killed the processing queue

## Steps To Reproduce
Has occurred twice with this particular video - oISaa0cJQgI

## Expected behavior
Work or verbalize an error and continue with other queued tasks

### Relevant log output

```shell
[2023-02-11 20:54:16,740: WARNING/ForkPoolWorker-32] [youtube] Extracted 532 comments
[2023-02-11 20:54:16,758: WARNING/ForkPoolWorker-32] [info] Testing format 248
[2023-02-11 20:54:21,643: WARNING/ForkPoolWorker-32] [info] Testing format 251
[2023-02-11 20:54:24,401: WARNING/ForkPoolWorker-32] [info] oISaa0cJQgI: Downloading 1 format(s): 248+251
[2023-02-11 20:54:24,458: ERROR/ForkPoolWorker-32] Task download_pending[d144760d-61c7-441e-a0d3-f0dca6cbfd73] raised unexpected: AttributeError("'NoneType' object has no attribute 'replace'")
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 102, in download_pending
    downloader.run_queue()
  File "/app/home/src/download/yt_dlp_handler.py", line 241, in run_queue
    DownloadPostProcess(self).run()
  File "/app/home/src/download/yt_dlp_handler.py", line 45, in run
    self.get_comments()
  File "/app/home/src/download/yt_dlp_handler.py", line 149, in get_comments
    CommentList(self.download.videos).index(notify=True)
  File "/app/home/src/index/comments.py", line 210, in index
    comment.build_json(notify=notify)
  File "/app/home/src/index/comments.py", line 39, in build_json
    self.format_comments(comments_raw)
  File "/app/home/src/index/comments.py", line 112, in format_comments
    cleaned_comment = self.clean_comment(comment)
  File "/app/home/src/index/comments.py", line 130, in clean_comment
    "comment_text": comment["text"].replace("\xa0", ""),
AttributeError: 'NoneType' object has no attribute 'replace'
```

**Comment by lamusmaser** [comment]
Confirmed that this does occur for video `oISaa0cJQgI`. Running through the process manually, this is the comment that fails:

```
{'id': 'Ugh5fejLNUxgtngCoAEC', 'text': None, 'timestamp': 1423699200, 'time_text': '8 years ago', 'like_count': 0, 'is_favorited': False, 'author': 'ree ziz', 'author_id': 'UCx-bSSQoGWpNjWFWwfpIUpA', 'author_thumbnail': 'https://yt3.ggpht.com/ytc/AL5GRJVccvOrJYRCfOXdUTIhk6j3C6r9jligA2nnLg=s176-c-k-c0x00ffffff-no-rj', 'author_is_uploader': False, 'parent': 'root'}
```

As noted, the `text` is `None`, which is causing the failure. It looks like there should be some kind of additional error handling in the `clean_comment()` method that allows it to detect if any of these are anomalous an replace them with a default entry. Mostly seems to be an assumption that each object attribute should be both present and of the valid type, and we need to work to validate those assumptions before putting them in the `cleaned_comment` variable.

**Comment by bbilly1** [comment]
Looks like the comment can be empty? Must be a glitch from YT's side, I don't think that's intentional. I'd say, just skip it. If a comment hasn't any text, there is no comment to save.

**Comment by bbilly1** [comment]
Thanks for reporting, this is now fixed in v0.3.3, skipping comments without content.
